cmd/gazelle/fix-update.go

```diff
@@ -70,7 +70,14 @@ func runFixUpdate(cmd command, args []string) error {
 	if err != nil {
 		return err
 	}
-	c.ShouldFix = cmd == fixCmd
+	if cmd == fixCmd {
+		c.ShouldFix = cmd == fixCmd
+
+		// Only check the version when "fix" is run. Generated build files
```

Comment:
If the version is wrong, what is the user path to making it right, and can we print an error message that tells them exactly what to do?
Ideally this would be a feature update-repos where it updates the version of rules_go in use as well?

Reply:
I've added a sentence to the message to update io_bazel_rules_go in WORKSPACE.

`update-repos` should be able to this in the future. It might need a special case, since rules_go isn't a normal Go repository, but the command could be the same as updating any other repository.